BoldGrid/crio
# Add 'published' or 'modified' post meta date control

Resolves #12 

[crio-2.19.0-issue-12.zip](https://github.com/BoldGrid/crio/files/10460241/crio-2.19.0-issue-12.zip)

### Testing Instructions ###

NOTE: This change is concurrent with a Crio Premium PR ( https://github.com/BoldGrid/crio-premium/pull/118 ). This is because there are two places where these controls are used ( Blog Page & Single Post view ). The Blog page control is free, but the Single Post controls are a Premium feature. Therefore these instructions are for the 'Blog Page' control. Testing instructions for the 'Single Post' control are the same ( except the control is located `Design > Blog > Posts > Links > Post Meta` ). To simplify testing, you may want to install both the Crio and Crio Pro test zips to test at the same time.

NOTE: If the 'modified on' and 'published on' dates are the same, the 'Posted on ....' or 'Posted .... ago' string will always show regardless of the setting of these controls.

1. Install Crio and activate starter content ( or install an inspiration with a blog, then replace with this zip ).
2. Modify one of your posts that is displayed on the blog page, and save.
     - NOTE: When editing the post, in order to have a noticeable difference between the Published and Modified date stamp, it may be helpful to change the 'Published On' date to an older date before saving. This can be done in the editor by clicking the 'edit' link above the 'Update' button, where the published date is shown.
4. Open the Customizer and navigate to `Design > Blog > Blog Page > Links > Post Meta`
5. In the Customizer preview, navigate to your Blog page.
6. Change the `Use Published or Modified Date` Control from 'Published' to 'Modified'
7. Ensure that the preview displays the proper string in the entry header of the post you modified earlier.
8. Publish customizer changes, then view the blog page live.
9. Ensure that the live view is correct.
10. Change the `Date Format` control, and repeat steps 6-9 for the alternate format type.
